src/components/Steps/index.js

```diff
@@ -14,108 +15,114 @@ class Steps extends React.Component {
         super(props);
         this.state = {
             steps: [
-                React.createRef(),
-                React.createRef(),
-                React.createRef(),
-                React.createRef(),
-            ],
-            map: false,
+                {
+                    name: 'Choose city',
+                    component: SelectCityStep,
+                },
+                {
+                    name: 'Choose your job/school address',
+                    component: SearchStep,
+                },
+                {
+                    name: 'Choose additional criteria',
+                    component: CriteriaStep,
+                },
+                {
+                    name: 'Map',
+                    component: MapStep,
+                },
+            ].map(step => ({
+                ...step,
+                ref: React.createRef(),
+                active: false,
+                errors: [],
+            })),
+            validated: false,
         };
+
+        this.activateStep(0);
     }
 
-    scrollTo =
-        ref => (ref.current
-            ? scrollToComponent(ref.current, {
-                offset: -50,
-                align: 'top',
-                duration: 1000,
-            })
-            : {});
+    componentDidUpdate() {
+        const lastActivatedStepIndex = this.state.steps.filter(step => step.active).length - 1;
```

Comment:
i would move steps to const

Reply:
I don't think that there's a need for that, since I have to specify that in state anyway ;0